internal/pkg/library/library.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package library
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/sylabs/scs-library-client/client"
+)
+
+const defaultTag = "latest"
+
+type progressCallback func(int64, io.Reader, io.Writer) error
+
+// NormalizeLibraryRef strips off leading "library://" prefix, if any, and
+// appends the default tag (latest) if none specified.
+func NormalizeLibraryRef(libraryRef string) string {
+	var ir string
+	if strings.HasPrefix(libraryRef, "library://") {
```

Comment:
This seems like a good place to use `strings.TrimPrefix()` instead

Reply:
You learn something new every day :) that's exactly what I need. Thanks!